pack.go

```diff
@@ -93,7 +93,9 @@ type PackManifestOptions struct {
 	// Layers is the layers of the manifest.
 	Layers []ocispec.Descriptor
 
-	// ManifestAnnotations is the annotation map of the manifest.
+	// ManifestAnnotations is the annotation map of the manifest. Set
+	// "org.opencontainers.image.created" with a value to make PackManifest
+	// reproducible.
```

Comment:
The timestamp must be in [RFC 3339](https://www.rfc-editor.org/rfc/rfc3339.html) format.

Reply:
Added a comment.